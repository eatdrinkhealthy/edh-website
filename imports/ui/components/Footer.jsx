import React from "react";

const Footer = () => (
  <footer className="app-footer">
    <div className="row">
      <div className="column">
        <img src="images/carrot_icon.png" alt="Company Logo" />
        <span className="app-footer__copyright">
          <span>Copyright 2018 Eat. Drink. Healthy.</span>
          <span>All rights reserved.</span>
        </span>
      </div>
    </div>
  </footer>
);

export default Footer;
