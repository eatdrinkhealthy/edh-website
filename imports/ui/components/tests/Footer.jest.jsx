/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
import React from "react";
import renderer from "react-test-renderer";
import Footer from "../Footer";

describe("<Footer />", () => {
  it("should match render snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
