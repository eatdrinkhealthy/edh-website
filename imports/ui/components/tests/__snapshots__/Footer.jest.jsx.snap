// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match render snapshot 1`] = `
<footer
  className="app-footer"
>
  <div
    className="row"
  >
    <div
      className="column"
    >
      <img
        alt="Company Logo"
        src="images/carrot_icon.png"
      />
      <span
        className="app-footer__copyright"
      >
        <span>
          Copyright 2018 Eat. Drink. Healthy.
        </span>
        <span>
          All rights reserved.
        </span>
      </span>
    </div>
  </div>
</footer>
`;
